pyproject.toml

```diff
@@ -22,3 +22,58 @@ environment = 'RUSTUP_TOOLCHAIN="stable"'
 [tool.cibuildwheel.linux]
 before-all = "yum install -y wget && {package}/tools/install_rust.sh"
 environment = 'PATH="$PATH:$HOME/.cargo/bin" CARGO_NET_GIT_FETCH_WITH_CLI="true" RUSTUP_TOOLCHAIN="stable"'
+
+
+[tool.ruff]
+line-length=105
+select = [
+  "E", # pycodestyle
+  "W", # pycodestyle
```

Comment:
Yes, we don't want `ruff` and `black` stepping on each others' toes. We can select or deselect them a la carte if this is enough to prevent conflicts.